index.template.html

```diff
@@ -5190,11 +5190,25 @@ <h3>Composition</h3>
     A single TD can also be generated which contains the interaction definitions of the top level/parent
     <a>Thing Model</a> and all interaction definitions of all sub/child <a>Thing Models</a>. 
     <span class="rfc2119-assertion" id="tm-compose-name-collision">
-       To avoid name collisions of the sub/child interaction names SHOULD rename the JSON name to the <code>instanceName</code> followed with <code>'_'</code> and the interaction name of the sub/child <a>Thing Model</a>.
+        Thereby the generation process MUST avoid possible name collisions. 
     </span> 
-    The following example shows a generated (self-contained) <a>Thing Description</a> of the Smart Ventilator Thing Model.
+    <span class="rfc2119-assertion" id="tm-compose-name-collision-seperation">
+        To achieve this, the following strategy is RECOMMENDED: 
+        All sub/child interaction names 
+        <ul>
+            <li>begin with the corresponding <code>instanceName</code> value,</li>
+            <li>followed with a predefined character(s) (e.g., '_', '.', '-', etc.), and</li>
```

Comment:
I would write "with a predefined character" (without plural) - as this is only a recommendation, you can keep it simple and recommend to use only one character.

Reply:
done